README.md

```diff
@@ -1,2 +1,494 @@
+
+  ```sh
+  docker-compose down
+  ```
+
+- Output
+
+  This run profile executes the entire test suite in parallel on a single browser, deployed on a docker image.
+
+- Note: By default, this execution would run maximum 5 test threads in parallel on Docker. Refer to the section ["Configuring the maximum parallel test threads for this repository"](#Configuring-the-maximum-parallel-test-threads-for-this-repository) for updating the parallel thread count based on your requirements.
+
+---
+
+# BrowserStack
+
+[BrowserStack](https://browserstack.com) provides instant access to 2,000+ real mobile devices and browsers on a highly reliable cloud infrastructure that effortlessly scales as testing needs grow.
+
+## Prerequisites
+
+- Create a new [BrowserStack account](https://www.browserstack.com/users/sign_up) or use an existing one.
+- Identify your BrowserStack username and access key from the [BrowserStack Automate Dashboard](https://automate.browserstack.com/) and export them as environment variables using the below commands.
+
+  - For \*nix based and Mac machines:
+
+  ```sh
+  export BROWSERSTACK_USERNAME=<browserstack-username> &&
+  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+  - For Windows:
+
+  ```shell
+  set BROWSERSTACK_USERNAME=<browserstack-username>
+  set BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+  Alternatively, you can also hardcode username and access_key objects in the [bs.json](resources/config/bs.json) file.
+
+Note:
+- We have configured a list of test capabilities in the [bs.json](resources/config/bs.json) file. You can certainly update them based on your device / browser test requirements.
+- The exact test capability values can be easily identified using the [Browserstack Capability Generator](https://browserstack.com/automate/capabilities)
+
+
+## Running Your Tests
+
+### Run a specific test on BrowserStack
+
+In this section, we will run a single test on Chrome browser on Browserstack. To change test capabilities for this configuration, please refer to the `single` object in `bs.json` file.
+
+- How to run the test?
+
+  - To run the default test scenario (e.g. End to End Scenario) on your own machine, use the following command:
+
+  Maven:
+  ```sh
+  mvn test -P bstack-single
+  ```
+
+  Gradle:
+    ```sh
+  gradle bstack-single
+  ```
+
+  To run a specific test scenario, use the following command with the additional 'test-name' argument:
+  Maven:
+  ```sh
+  mvn test -P bstack-single -Dtest-name="<Test scenario name>"
+  ```
+
+  Gradle:
+  ```sh
+  gradle bstack-single -Dtest-name="<Test scenario name>"
+  ```
+
+  where,  the argument 'test-name' can be any Cucumber scenario name configured in this repository.
+
+  E.g. "Login as username", "Login as Locked User", "Offers for mumbai geo-location" or any of the other test scenario names, as outlined in [About the tests in this repository](#About-the-tests-in-this-repository) section.
```

Comment:
`mvn test -P on-prem -Dtest-name="Offers for mumbai geo-location"`
`mvn test -P on-prem -Dtest-name="Login as username"`

Both these profiles do not trigger any test.

Reply:
The profile names are different than the one's mentioned in the README.md
This is the right command:
 `mvn test -P on-prem -Dtest-name="Login with given '<username>' user" `
` mvn test -P on-prem -Dtest-name="Offers for Mumbai location"`        
We will make the changes.